index.html

```diff
@@ -0,0 +1,158 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Product Landing Page</title>
+    <link rel="stylesheet" href="./style.css">
+
+    <!-- Latest compiled and minified CSS -->
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.2/dist/css/bootstrap.min.css">
+
+    <style>
+        /* Stackoverflow preview fix, please ignore */
+        
+        .navbar-nav {
+            flex-direction: row;
+        }
+        
+        .nav-link {
+            padding-right: .5rem !important;
+            padding-left: .5rem !important;
+        }
+    </style>
+
+</head>
+
+<body>
+
+    <div class="container">
+        <header id="header">
+
+
+            <nav class="navbar navbar-expand-lg navbar-light bg-light" id="nav-bar">
+                <a class="navbar-brand" href="#"><img src="" alt="" id="header-img">Logo</a>
+                <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
+                  <span class="navbar-toggler-icon"></span>
+                </button>
```

Comment:
- This button section is performing no action in the project since it should be removed.

Reply:
- [ ] You can also offer your opinion as a suggestion. example: I suggest this button should be removed because it is performing no action in the project, what do you think?